src/components/PreviousExamsTable/index.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+
+import User from 'src/types/User';
+import Pagination from '../Pagination';
+import Link from 'next/link';
+import { formatDateDDStrMonthYYYY } from 'src/utils/Date';
+import { useEffect, useRef, useState } from 'react';
+
+interface PreviousExamsTableProps {
+  user: User;
+}
+
+export default function PreviousExamsTable({ user }: PreviousExamsTableProps) {
```

Comment:
Change the fetch to inside this component.

The backend will need to change and probably a new route will be created that lists all the user exams something on the lines of: `/exams? page=x&items=y'